Src/BootCamp.Chapter/UserInteraction.cs

```diff
@@ -0,0 +1,207 @@
+            Console.WriteLine("(3) Crack encrypted message");
+            Console.WriteLine();
+            Console.WriteLine("(9) Settings");
+            Console.WriteLine("(0) Exit program!");
+
+            ChooseMenu();
+        }
+
+        private static void ChooseMenu()
+        {
+            ConsoleKey pressedKey = Console.ReadKey(true).Key;
+            if (pressedKey == ConsoleKey.D0)
+            {
+                return;
+            }
+            else if (pressedKey == ConsoleKey.D1)
+            {
+                EncryptMessage();
+            }
+            else if (pressedKey == ConsoleKey.D2)
+            {
+                DecryptMessage();
+            }
+            else if (pressedKey == ConsoleKey.D3)
+            {
+                CrackMessage();
+            }
+            else if (pressedKey == ConsoleKey.D9)
+            {
+                ChangeSettings();
+            }
+            else
+            {
+                Console.WriteLine();
+                Console.WriteLine("Invalid option!");
+                Wait();
+            }
+            DisplayMenu();
+        }
+
+        private static void ChangeSettings()
+        {
+            ConsoleInit(true);
+
+            DisplayHeader();
+
+            Console.WriteLine($"Key accuracy: {CaesarCipher.KeyAccuracy}");
+            Console.WriteLine($"Base character: {CaesarCipher.BaseCharacter}");
+            Console.WriteLine($"Top character: {CaesarCipher.TopCharacter}");
+
+            Console.WriteLine();
+
+            Console.Write("Do you want to change? (Y/N) ");
+            var response = Console.ReadLine();
+            if (response != "Y")
+            {
+                return;
+            }
+
+            UpdateKeyAccuracy();
+            UpdateBaseCharacter();
+            UpdateTopCharacter();
+
+            Console.WriteLine("Settings updated.");
+            Wait();
+        }
+
+        private static void UpdateBaseCharacter()
+        {
+            Console.Write("Input base character: ");
+            var baseCharacter = Console.ReadLine();
+            if (!Utils.IsNumeric(baseCharacter))
+            {
+                return;
+            }
+            CaesarCipher.BaseCharacter = Utils.ConvertToInt(baseCharacter);
+        }
+
+        private static void UpdateTopCharacter()
+        {
+            Console.Write("Input top character: ");
+            var topCharacter = Console.ReadLine();
+            if (!Utils.IsNumeric(topCharacter))
+            {
+                return;
+            }
+            CaesarCipher.TopCharacter = Utils.ConvertToInt(topCharacter);
+        }
+
+        private static void UpdateKeyAccuracy()
+        {
+            Console.Write("Input key accuracy: ");
+            var keyAccuracy = Console.ReadLine();
+            if (!Utils.IsNumeric(keyAccuracy))
+            {
+                return;
+            }
+            CaesarCipher.KeyAccuracy = Utils.ConvertToInt(keyAccuracy);
+        }
```

Comment:
This could be one function with return type int and taking parameters of type string and int.